javalib/src/main/scala/java/util/package.scala

```diff
@@ -33,4 +35,186 @@ package object util {
     }
   }
 
+  private[util] class NavigableView[E](original: NavigableSet[E],
```

Comment:
This class should be in its own file. There is no reason to put it in the package object.